aiida/orm/nodes/node.py

```diff
@@ -901,6 +910,9 @@ def get_incoming(self, node_class=None, link_type=(), link_label_filter=None):
                 raise exceptions.InternalError('Node<{}> has both a stored and cached link triple {}'.format(
                     self.pk, link_triple))
 
+            if only_uuid:
```

Comment:
Shouldn't this go above? the `link_triples` from the DB already have only UUIDs

Reply:
correct, will change it